crates/y-sweet-core/src/api_types.rs

```diff
@@ -66,6 +59,10 @@ pub struct ClientToken {
     /// An optional token that can be used to authenticate the client to the server.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub token: Option<String>,
+
+    /// The authorization level of the client.
+    #[serde(rename = "authorization")]
+    pub authorization: Authorization,
```

Comment:
We're now returning `authorization` as part of the `ClientToken` so that clients can limit front-end functionality (or just render differently) when the token is `read-only`.